packages/cli/__tests__/help/__integration__/__snapshots__/BrightHelpTests.integration.test.ts.snap

```diff
@@ -34,6 +34,7 @@ exports[`Root level help tests top level help should contain support link 1`] =
                                  responses                                    
    zos-files | files             Manage z/OS data sets                        
    zos-jobs | jobs               Manage z/OS jobs                             
+   zos-logs | zoslogs            Inteact with z/OS logs                       
```

Comment:
I'd suggest to change the alias (second name) from "zoslogs" to just "logs", to be consistent with other command groups.

Reply:
Hi @t1m0thyj , thank you very much for all the reviews and suggestions!
And thanks @Joey-ibm for clarifying why we use zos-logs as group.

We had some discussion in  [Slack channel](https://openmainframeproject.slack.com/archives/CC8AALGN6/p1631265196013100?thread_ts=1631159427.009900&cid=CC8AALGN6) about the 4-levels command name, and why we use zoslogs as alias:

`We use 2nd level group specific for zos-logs, and we will alias it with zoslogs so that other types of logs like z/OSMF and DB2 can use different 2nd level group if needed. `

Please let me know if you have any concern, thank you!